website/versioned_docs/version-6.26.3/register.md

```diff
@@ -101,3 +101,4 @@ Disable the cache.
 BABEL_DISABLE_CACHE=1 babel-node script.js
 ```
 
+**Note:** `@babel/register` currently does not support compiling native Node.js ES modules on the fly. This is because Node.js doesn't have a stable API yet to intercept ES modules loading.
```

Comment:
Since this is for Babel 6, it should be `babel-register`.

Reply:
Made the changes. PTAL.